txsni/snimap.py

```diff
@@ -44,6 +49,28 @@ def negotiateALPN(self, context):
         context.set_alpn_protos(self.alpnProtocols)
 
 
+def _detect_acme(buf):
+    """
+    Determine whether buf is probably acme-tls/1
+
+    Example ClientHello from letsencrypt
+
+    b'\x16\x03\x01\x00\xe9\x01\x00\x00\xe5\x03\x03r\xfaO\xbb\xd5\xbf\x9fh'\
```

Comment:
This feels unnecessary and the sort of thing which is going to trip false-positives on scanners looking for secrets in public git repos.  Can we eliminate it and replace it with an explanation of what's going on?

Reply:
Specifically: this *should* be parsing the TLS message, which is actually not as hard as it sounds - we don't need to look at any ASN.1 to do this right, just a bunch of length prefixes.